Android/firebase_dependencies.gradle

```diff
@@ -17,15 +17,15 @@ import org.gradle.util.ConfigureUtil;
 // A map of library to the dependencies that need to be added for it.
 def firebaseDependenciesMap = [
   'app' : ['com.google.firebase:firebase-analytics:19.0.0'],
-  'admob' : ['com.google.firebase:firebase-ads:19.8.0',
+  'admob' : ['com.google.firebase:firebase-ads:20.2.0',
```

Comment:
You'll want to leave all the ads (for both Android and iOS) pinned at their old version.